extensions/blocks/slideshow/transforms.js

```diff
@@ -20,9 +20,9 @@ const transforms = {
 			type: 'block',
 			isMultiBlock: true,
 			blocks: [ 'core/image' ],
-			isMatch: attributes => getValidImages( attributes ).length > 0,
-			transform: attributes => {
-				const validImages = getValidImages( attributes );
+			isMatch: images => getValidImages( images ).length > 0,
+			transform: images => {
+				const validImages = getValidImages( images );
```

Comment:
These are simple variable name changes — it was confusing to see `attributes` in place of `images` where in the rest of the transforms we have `attributes.images` (in form of `( { images } )`).

There are a few similar ones across the changeset.

Reply:
I appreciate this. It makes it far more clearer what is being passed between the functions.